index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="styles.css">
```

Comment:
Great Job adding the HTML file.

[OPTIONAL] 😉 Please consider adding some other useful information to your `index.html` head section. A title would be amazing and would make your page look more professional. Also, consider adding some meta tags like the charset, description, or viewport meta tag. This is not required but making this a habit would make you a better dev.

Reply:
Hi, @rskinnerc, I have added the title and some meta tags in the header section of the index.html file. Please go through it and let me know if you need any changes.